english/teams/team-captain.md

```diff
@@ -0,0 +1,2 @@
+Team Captain America believes that if a governing body is allowed to dictate terms to superheroes, they could be misused to serve
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.